app/api/microservice_info.py

```diff
@@ -0,0 +1,29 @@
+from fastapi import APIRouter, Depends
```

Comment:
please take a look on the imports and remove the unnecessary ones.

Reply:
done